getting-started-with-api/java-sdk/src/main/java/Upload.java

```diff
@@ -0,0 +1,75 @@
+import com.smartling.api.files.v2.pto.FileType;
+import com.smartling.api.files.v2.pto.UploadFilePTO;
+import com.smartling.api.files.v2.pto.UploadFileResponse;
+import com.smartling.api.v2.authentication.AuthenticationApi;
+import com.smartling.api.v2.authentication.AuthenticationApiFactory;
+import com.smartling.api.v2.client.ClientFactory;
+import com.smartling.api.v2.client.DefaultClientConfiguration;
+import com.smartling.api.v2.client.auth.Authenticator;
+import com.smartling.api.v2.client.auth.BearerAuthSecretFilter;
+import com.smartling.api.v2.client.exception.RestApiRuntimeException;
+
+import java.io.FileInputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Upload
+{
+    private static final List<String> locales = new ArrayList<>(Arrays.asList("fr-FR"));
+
+    public static void main( String[] args ) throws Exception
+    {
+        // Read command-line arguments
+        if (args.length != 2) {
+            System.out.println("Usage: mvn -X exec:java -Dexec.mainClass=\"Upload\" -Dexec.args=\"<fileName> <fileType>\"");
+            System.out.println("E.g.: mvn -X exec:java -Dexec.mainClass=\"Upload\" -Dexec.args=\"strings.json json\"");
+            System.out.println("Takes two command-line parameters: ");
+            System.out.println("  fileName: local name of file to be uploaded");
+            System.out.println("  fileType: file type of file to be uploaded");
+            System.exit(1);
+        }
+        String fileName = args[0];
+        String fileType = args[1];
+
+        // Read authentication credentials from environment
+        String userId = System.getenv("DEV_USER_IDENTIFIER");
+        String userSecret = System.getenv("DEV_USER_SECRET");
+        String projectId = System.getenv("DEV_PROJECT_ID");
+        if (userId == null || userSecret == null || projectId == null) {
+            System.out.println("Missing environment variables. Did you run setenv?");
+            System.exit(1);
+        }
+
+        // Create factory for building API clients
+        DefaultClientConfiguration clientConfiguration = DefaultClientConfiguration.builder().build();
+        ClientFactory clientFactory = new ClientFactory();
+
+        // Create the authenticator to be used by subsequent API calls
+        AuthenticationApi authenticationApi = new AuthenticationApiFactory(clientFactory).buildApi(clientConfiguration);
+        Authenticator authenticator = new Authenticator(userId, userSecret, authenticationApi);
+
+        // Create files API client
+        FilesApi filesApi = new FilesApiFactory(clientFactory)
+            .buildApi(new BearerAuthSecretFilter(authenticator), clientConfiguration);
+
+        // Build the API request
+        UploadFilePTO uploadFilePto = UploadFilePTO.builder()
+            .file(new FileInputStream(fileName))
+            .fileUri(fileName)
+            .fileType(FileType.lookup(fileType))
+            .localeIdsToAuthorize(locales)
```

Comment:
@conall-smartling I don't think that we would like to advertise this param. It's already obsolete. I see the following options:
- Tell that you can upload source content only and user should authorize it manually (by adding the file to job)
- Or make example slightly complex by using Jobs and Job Batches SDKs

Reply:
Yes, I agree. I'll remove it from this example. It's not needed anyway since we're only downloading pseudo translations. However, I think I might also do the slightly more complex example with jobs and batches. Even though that type of example might not be a realistic integration (in the way that SpringBoot might be), it still would help someone who's playing with and getting familiar with the APIs and Smartling.